here/enums.go

```diff
@@ -64,15 +65,16 @@ var ReverseGeocodingMode = &reverseGeocodingList{
 }
 
 // ValueOfReverseGeocodingMode returns value for ReverseGeocodingMode.
-func (reverseGeocodingMode Enum) ValueOfReverseGeocodingMode() string {
+func (routeMode Enum) ValueOfReverseGeocodingMode() string {
```

Comment:
I think it's better to update `routeMode` on `func (routeMode Enum) ValueOfRouteMode()` and on `func (routeMode Enum) ValueOfReverseGeocodingMode()` as `mode`.

Reply:
I think it's done. note: I'm doing it in the github editor, so there might be some typos yet

Reply:
Looks good now thanks.